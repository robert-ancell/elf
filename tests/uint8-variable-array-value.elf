uint8 v = [1, 2, 3]
