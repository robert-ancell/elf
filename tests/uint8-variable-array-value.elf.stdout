Line 1:
uint8 v = [1, 2, 3]
      ^
Variable is of type uint8, but value is of type uint8[]
